lib/MusicBrainz/DataStore/Redis.pm

```diff
@@ -48,6 +48,14 @@ sub get {
     return defined $value ? $self->_json->decode($value) : undef;
 }
 
+sub mget {
+    my ($self, @keys) = @_;
+
+    map {
+        defined $_ ? $self->_json->decode($_) : ()
```

Comment:
Argh. It was working locally because I had alert, alert_mtime keys in redis, so the array got shifted enough. Thanks for finding that!